database/sites.go

```diff
@@ -55,10 +55,10 @@ func GetSitesOfUser(user models.User) ([]models.Site, error) {
 	var sites []models.Site
 	err := DB.C(sitesCollection).Find(bson.M{
 		"$or": []bson.M{
-			bson.M{
+			{
```

Comment:
This code change is exec `gofmt -s -w ./...` auto optimized.  
Can use `make fmt` direct call this cmd.